src/errors/test-run/shared.js

```diff
@@ -0,0 +1,399 @@
+// -------------------------------------------------------------
```

Comment:
Not sure about the name of this file. These file contains only information about errors. Even if the file is used on server and client sides we already have a note in the beginning of file

Reply:
Ideally, we should have a dedicated directory for `shared` scripts. We can try to start with `/src/shared/errors`.